Sources/tart/Commands/Prune.swift

```diff
@@ -103,5 +104,7 @@ struct Prune: AsyncParsableCommand {
     }
 
     puppy.info("reclaimed \(cacheReclaimedBytes) bytes")
+
+    SentrySDK.span?.setMeasurement(name: "gc_disk_reclaimed", value: cacheReclaimedBytes as NSNumber, unit: MeasurementUnitInformation.byte);
```

Comment:
I think you need to [bind the transaction to scope](https://docs.sentry.io/platforms/apple/performance/instrumentation/custom-instrumentation/#create-transaction-bound-to-the-current-scope) the first, in order for the `SentrySDK.span` to work.

Reply:
My bad. I saw the documentation and though it's binded by default. 😪 Good catch!